### PR TITLE
Yadda v1

### PR DESCRIPTION
**Pull request by djkotowski**

### CODE REVIEW

**Comment by jmmastey** [comment]
Can you submit a pull for the relevant hound and gitignore stuff so we can include it upstream?

File: yadda/populate/.bundle/config [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_JOBS: 8
```
Comment (jmmastey) [inline yadda/populate/.bundle/config]: Oh yeah?

**Comment by djkotowski** [comment]
I'll take a look at the hound file.  It looks like there's some duplication from a merge.

File: yadda/populate/.bundle/config (hunk 1 above)
Comment (djkotowski) [inline yadda/populate/.bundle/config]: I'll ensure `.bundle/` is excluded from Git in the future, given these configs are supposed to be machine-specific.

**Comment by jmmastey** [comment]
This is all good :)

**Pull request closed without merging**